shacl12-core/index.html

```diff
@@ -1832,8 +1828,8 @@ <h4>Declaring the Severity of a Shape or Constraint</h4>
 					</span></p>
 					<p><em>The remainder of this section is informative.</em></p>
 					<p>
-						The specific values of <code>sh:severity</code> have no impact on the validation,
-						but MAY be used by user interface tools to categorize validation results.
+						The validation handles the values of <code>sh:severity</code> according to the <a>conformance checking</a>.
```

Comment:
Non-blocking grammar revision:

```suggestion
						The validation process handles the values of <code>sh:severity</code> according to <a>conformance checking</a>.
```

"The validation process" is used elsewhere in the document.

I think "The conformance checking" should either drop "the" or end "configuration," "setting" or similar.